bot/cogs/candy_collection.py

```diff
@@ -0,0 +1,209 @@
+import discord
+from discord.ext import commands
+import random
+import json
+import functools
+import os
+
+json_location = os.path.join(os.getcwd(), 'resources', 'candy_collection.json')
+
+
+class CandyCollection:
+    def __init__(self, bot):
+        self.bot = bot
+        with open(json_location) as candy:
+            self.candy_json = json.load(candy)
+            self.msg_reacted = self.candy_json['msg_reacted']
+        self.get_candyinfo = dict()
+        for userinfo in self.candy_json['records']:
+            userid = userinfo['userid']
+            self.get_candyinfo[userid] = userinfo
+
+    HACKTOBER_CHANNEL_ID = 498804484324196362
```

Comment:
These constants should be at the top of the file underneath the imports, not inside the class.